derrick

```diff
@@ -48,6 +48,9 @@ case "$1" in
     "test:js")
         dcrq yarn test:js "${@:2}"
         ;;
+    "storybook")
+        dcrq yarn run storybook "${@:2}"
+        ;;
```

Comment:
@jgerigmeyer Can you point out what's missing for this work. The idea is to be able to run storybook from the container, but I feel like there may be another set needed here.    
`$ ./derrick storybook
yarn run v1.22.5
$ start-storybook -p 6006 
/bin/sh: 1: start-storybook: not found
`

Reply:
@duggiemitchell Fixed!